frameworks/g-cloud-9/questions/services/freeVersionTrialOption.yml

```diff
@@ -1,6 +1,6 @@
 name: Trial option
 question: Do you provide a free trial option for your service?
-
+filter_label: free trial
```

Comment:
Can we keep a linebreak after the filter_label so they stay in four separate blocks, cheers... :pedant: